src/pages/painting/[slug].js

```diff
@@ -39,6 +44,24 @@ export async function getStaticProps({ params }) {
     name: paintingResponse.collection?.name,
   });
 
+// If theres 1 painting in the collection we will remove the current painting from the recommended
```

Comment:
Ohh ok I understandm thanks for explaining. How do you feel about this? I am just trying to make it a little DRYer:
```
let paintingsToReccommend = [];
  if (collectionResponse.paintingsCollection.items.length <= 1) {
    paintingsToReccommend = await getAllPaintings();
  } else {
    paintingsToReccommend = collectionResponse.paintingsCollection.items;
  }

  const recommendedPaintings = removeCurrentPaintingFromRecommendedList(
    paintingResponse.name,
    paintingsToReccommend
  );```